x-pack/plugins/cases/server/client/cases/find.test.ts

```diff
@@ -114,5 +119,35 @@ describe('find', () => {
         `Error: Too many categories provided. The maximum allowed is ${MAX_CATEGORY_FILTER_LENGTH}`
       );
     });
+
+    it(`throws an error when the tags array has ${MAX_TAGS_FILTER_LENGTH} items`, async () => {
+      const tags = Array(MAX_TAGS_FILTER_LENGTH + 1).fill('foobar');
+
+      const findRequest = createCasesClientMockFindRequest({ tags });
+
+      await expect(find(findRequest, clientArgs)).rejects.toThrowErrorMatchingInlineSnapshot(
```

Comment:
I know we typically do matching inline snapshot. That's a lot of foobars though haha. Maybe we just check that it throws an error? Or check that the error contains `Error: array must be of length <= 100`?

Reply:
I agree, this is getting out of hand 😄